frontend/src/metabase/palette/hooks/useCommandPalette.tsx

```diff
@@ -88,48 +91,50 @@ export const useCommandPalette = ({
     showDocsAction,
   ]);
 
-  const searchResultActions = useMemo<KBarAction[]>(() => {
-    const ret: KBarAction[] = [];
+  const searchResultActions = useMemo<PaletteAction[]>(() => {
+    const ret: PaletteAction[] = [];
```

Comment:
!nit, rather than pushing to an array, can't you return values directly from each conditional branch? not a big deal, but this pattern made me assume that somewhere later on another condition would push more values.